src/ui/simulator/application/study.cpp

```diff
@@ -393,7 +393,14 @@ class JobSaveStudy final : public Toolbox::Jobs::Job
         }     // save as
 
```

Comment:
This is obviously code duplication. Do you have proposal where potential function should be defined?